File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.py
def solution(numbers):
    max_num = numbers[0] * numbers[1]
    for num1 in numbers:
        for num2 in numbers:
            if num1 != num2 and num1 * num2 > max_num:
                max_num = num1 * num2
    return max_num

File: 프로그래머스/0/120000/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.java
import java.util.*;

class Solution {
    public int solution(int[] numbers) {
        Arrays.sort(numbers);
        int max_num1 = numbers[numbers.length - 1] * numbers[numbers.length - 2];
        int max_num2 = numbers[1] * numbers[0];
        return Math.max(max_num1 , max_num2);
    }
}